ui/pages/confirmations/confirm/confirm.tsx

```diff
@@ -20,7 +20,7 @@ const Confirm = () => {
       <Content backgroundColor={BackgroundColor.backgroundAlternative}>
         <Title />
         <Info />
-        <Box>CONFIRMATION PAGE BODY TO COME HERE</Box>
+        <SignatureMessage />
```

Comment:
Should this be abstracted within the `Info` component? I would expect that the different info blocks would be coupled to each confirmation info component. For example, `PersonalSignInfo` component should return the actual structure that we are expecting:
```js
<>
  <Box
    ...
  >
    <ConfirmInfoRow>
     ...
    </ConfirmInfoRow>
    ...
  </Box>
  <SignatureMessage>
</>
```
And the Info component could just be confirmation map to the right infoComponent and return it:
```js
  return (
    <InfoComponent />
  )
```

What do you think about it?

Reply:
Yes I also have been thinking about this, I actually I wanted to discuss the confirmation designs with Saya to have more clarity on this. In one of the following PRs I will make this change.